apps/frontend/src/containers/root.tsx

```diff
@@ -1,10 +1,15 @@
 import { Outlet } from 'react-router-dom';
-import Header from '../../components/Header';
+import Header from '@components/Header';
+import { SSFLoginForm, PantryLoginForm } from '@components/forms/loginForm';
+import { useState } from 'react';
 const Root: React.FC = () => {
+  const [ssf, setSSF] = useState(false);
   return (
     <div>
       <Header />
       <Outlet />
+      {ssf ? <SSFLoginForm /> : <PantryLoginForm />}
```

Comment:
Assuming this PR corresponds to [this ticket](https://trello.com/c/J83kEdOu), should we be adding this to `/landing-page` instead? Right now, the login forms show up on every page on the site because all pages show the root content.

Reply:
goot catch sam i have moved it to the landing page